aws-globalaccelerator-accelerator/aws-globalaccelerator-accelerator.json

```diff
@@ -73,6 +73,18 @@
         "type": "string"
       }
     },
+    "Ipv6Addresses": {
+      "description": "The IPv6 addresses assigned if the accelerator is dualstack",
+      "default": null,
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "DualStackDnsName": {
+      "description": "The Domain Name System (DNS) name that Global Accelerator creates that points to your accelerator's static IPv6 addresses.",
```

Comment:
Let's update this to be both IPv4 & IPv6 and then fix up the previous to call out just IPv6

Reply:
Done